apps/api/src/graphql/resolvers/courses.ts

```diff
@@ -20,5 +20,15 @@ export const coursesResolvers = {
       const service = new CoursesService(db);
       return await service.batchGetCourses(ids);
     },
+    coursesByCursor: async (_: unknown, args: { query: unknown }, { db }: GraphQLContext) => {
```

Comment:
Try to keep the order of the resolvers in line with the order of the underlying functions on the service.